assets/sass/protocol/includes/_lib.scss

```diff
@@ -2,49 +2,13 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-//* -------------------------------------------------------------------------- */
-// Brand theme
-
-// Protocol supports two distinct brands - Firefox and Mozilla
-// The brand themes draw from different sets of variables to
-// apply different styling to some elements based on which brand
-// you're representing on the page.
-
-// This sets Mozilla as the default brand out of the box. You can
-// override this variable in other style sheets by setting a different
-// value *before* the lib file import.
-
-// Accepted values:
-//      'firefox'
-//      'mozilla'
-
-$brand-theme: 'mozilla' !default;
-
-
-//* -------------------------------------------------------------------------- */
-// Type scale
-
-// Protocol uses a modular type scale with a range of predefined
-// font sizes (to avoid sizing text willy nilly). Scale sizes are
-// defined as part of the brand theme.
-
-// There are two different scales available: a standard scale with
-// large headings, appropriate for general websites (like marketing
-// sites, blogs, etc) and a condensed scale with smaller sizes more
-// suitable for web applications.
-
-// You can override this variable in other style sheets by setting a
-// different value *before* the lib file import.
-
-// Accepted values:
-//     'standard'
-//     'condensed'
-
-$type-scale: 'standard' !default;
-
+// Forward user-configurable variables
+@forward 'config';
+// Namespace theme functions
+@forward 'themes';
 
 // Import all the things!
-@import './node_modules/@mozilla-protocol/tokens/dist/index.scss';
-@import 'functions';
-@import 'themes';
-@import 'mixins';
+@forward '../../../../node_modules/@mozilla-protocol/tokens/dist/index.scss';
```

Comment:
This was the first error I hit, where the sass migration tool failed because it couldn't find the node modules import, so it was a de-blocking change. Definitely a good idea to avoid the shaky relative path in a permanent solution.

Reply:
Curious if webpack will through an error during compilation, since that file won't be available until _after_ compilation?

Reply:
Static files are copied over prior to the Sass build, so should not be a problem I think (but haven't checked)